### ISSUE TITLE
Crash at start-time

### ISSUE THREAD

**Issue by arcuri82** [JetBrains/idea-gitignore#615]
### Prerequisites

* [x] Plugin is in the latest version
* [x] Issue was not reported yet
* [x] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Description

Starting IDEA, the plugin crashes with: 

```
Plugin 'mobi.hsz.idea.gitignore' failed to initialize and will be disabled.  Please restart IntelliJ IDEA.

java.lang.IllegalAccessError: failed to access class com.intellij.util.containers.ConcurrentWeakHashMap from class mobi.hsz.idea.gitignore.util.CachedConcurrentMap (com.intellij.util.containers.ConcurrentWeakHashMap is in unnamed module of loader com.intellij.util.lang.UrlClassLoader @39fcbef6; mobi.hsz.idea.gitignore.util.CachedConcurrentMap is in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @4a4d3f54)
    at mobi.hsz.idea.gitignore.util.CachedConcurrentMap.<init>(CachedConcurrentMap.java:53)
    at mobi.hsz.idea.gitignore.util.CachedConcurrentMap.create(CachedConcurrentMap.java:65)
    at mobi.hsz.idea.gitignore.IgnoreBundle.<clinit>(IgnoreBundle.java:110)
    at mobi.hsz.idea.gitignore.file.IgnoreFileTypeFactory.createFileTypes(IgnoreFileTypeFactory.java:50)
    at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.lambda$initStandardFileTypes$0(FileTypeManagerImpl.java:360)
    at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processWithPluginDescriptor(ExtensionPointImpl.java:292)
    at com.intellij.openapi.extensions.ExtensionPointName.processWithPluginDescriptor(ExtensionPointName.java:157)
    at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.initStandardFileTypes(FileTypeManagerImpl.java:358)
    at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.<init>(FileTypeManagerImpl.java:228)
    at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
    at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:44)
    at com.intellij.serviceContainer.PlatformComponentManagerImpl.instantiateClassWithConstructorInjection(PlatformComponentManagerImpl.kt:505)
    at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:52)
    at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:39)
    at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:110)
    at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:72)
    at com.intellij.serviceContainer.BaseComponentAdapter.getInstance$default(BaseComponentAdapter.kt:65)
    at com.intellij.serviceContainer.PlatformComponentManagerImpl$preloadServices$future$1.run(PlatformComponentManagerImpl.kt:622)
    at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
    at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
    at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:30)
    at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
    at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:193)
    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
    at java.base/java.lang.Thread.run(Thread.java:834)
```

### Steps to Reproduce

1. It happens at start-time

**Expected behavior:**

No crash

**Actual behavior:** 

It crashes

**Reproduces how often:** 

Every single time I start IDEA, since I updated to latest version

### Versions

**Plugin:** 
```
2.6.2
```

**IDE:** 
```
IntelliJ IDEA 2019.3.1 (Ultimate Edition)
Build #IU-193.5662.53, built on December 18, 2019
Runtime version: 11.0.5+10-b520.17 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 8
Registry: ide.editor.tabs.open.at.the.end=true
Non-Bundled Plugins: ASM Bytecode Outline, BashSupport, PIT mutation testing Idea plugin, org.jetbrains.plugins.rest, org.jetbrains.plugins.localization, Pythonid, R4Intellij, YourKit Java Profiler 2017.02-b63 Integration, com.intellij.lang.jsgraphql, net.seesharpsoft.intellij.plugins.csv, org.antlr.intellij.plugin, org.asciidoctor.intellij.asciidoc```
```

**OS:** 

```
macOS 10.14
MacBook Pro (Retina, 15-inch, Mid 2014)
```

**Comment by inforithmics** [comment]
Had a similar problem uninstalling and reinstalling solved it.

**Comment by hsz** [comment]
Fixed with `v4.0.0`.
